README.md

```diff
@@ -28,6 +36,7 @@ The site is optimized for GPS-enabled cameras.  The majority of smartphones and
 3. [Feature Description] - Allow users to manually specify the location info for photos without embedded EXIF data
```

Comment:
I'm guessing "[Feature Description]" was probably a placeholder that we can delete in these lines, right?

Reply:
Actually, no.  I missed adding the ` formatter to the markdown.  It's supposed format the "Feature Description" text like this: `Feature Description`.  I'll add it now unless you'd rather just remove it altogether.